### PR TITLE
Delete some dead code and fix nits.

### PR DESCRIPTION
**Pull request by monopole**
Nits == package name used as a var name, or as a camelcase prefix to a var name

### CODE REVIEW

File: pkg/commands/build/build.go [hunk 1]
```diff
@@ -29,20 +29,12 @@ import (
...
-// BuildOptions contain the options for running a build
-type BuildOptions struct {
+// Options contain the options for running a build
+type Options struct {
 	kustomizationPath string
 	outputPath        string
 }
 
-// NewBuildOptions creates a BuildOptions object
-func NewBuildOptions(p, o string) *BuildOptions {
```
Comment (Liujingfang1) [inline pkg/commands/build/build.go]: Please don't remove this function. We need it in kubectl to create a `BuildOptions` object. 
```
o := NewBuildOptions(path, "")
o.RunBuild()
```

Comment (monopole) [reply pkg/commands/build/build.go]: ah yeah.

 can we call it build.Options instead of build.BuildOptions?

Comment (Liujingfang1) [reply pkg/commands/build/build.go]: Yes, go for it.

Comment (monopole) [reply pkg/commands/build/build.go]: To clear the linter errors.

Comment (monopole) [reply pkg/commands/build/build.go]: oky doky, done

**Pull request merged**